packages/resilience/src/middleware.ts

```diff
@@ -130,7 +130,7 @@ function addMiddlewaresToInitialFunction<
   ContextT extends MiddlewareContext<ArgumentT>
 >(
   middlewares: Middleware<ArgumentT, ReturnT, ContextT>[],
-  initial: MiddlewareIn<ArgumentT, ReturnT, ContextT>
+  initial: MiddlewareOptions<ArgumentT, ReturnT, ContextT>
 ): MiddlewareOut<ArgumentT, ReturnT> {
```

Comment:
[q] the return type here can also be changed to `MiddlewareFunction` right?

Reply:
yes removed them all